src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -171,8 +175,25 @@ def create_job_from_job_bundle(
     }
 
     storage_profile_id = get_setting("settings.storage_profile_id", config=config)
+    storage_profile = None
     if storage_profile_id:
         create_job_args["storageProfileId"] = storage_profile_id
+        storage_profile_response = deadline.get_storage_profile_for_queue(
```

Comment:
Nit: since we're doing this in two places, we might want to make a function wrapper like what we've done in other places, like queue params https://github.com/aws-deadline/deadline-cloud/blob/mainline/src/deadline/client/api/_queue_parameters.py

Not blocking though, since we didn't hit the rule of 3 ;)

Reply:
Thank you for the advise. I have dded a function wrapper `get_storage_profile_for_queue` in client/api/_get_storage_profile_for_queue.py.